### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by idcuadrowolox**
## Summary
- Footer

## Screenshot
screencapture-file-Users-ivandariocuadroguzman-ic-angular-maquetado-index-html-2019-04-03-18_40_31

## Trello Card
https://trello.com/c/cG6y68au/14-footer

### CODE REVIEW

File: maquetado/scss/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  width: 100%;
+
+  &-copyright {
+    align-self: flex-end;
+    color: inherit;
+    font-size: $text-small;
+  }
+
+  &-logo {
+    height: 60px;
```
Comment (henryzarza) [inline maquetado/scss/_footer.scss]: Invert the order, this value (`60px`) assign it to `width` property and the `height` assign it `auto` value

**Review by henryzarza — CHANGES_REQUESTED** [review]
Excellent job 👏 , only one change and let's go!

File: maquetado/scss/_footer.scss (hunk 1 above)
Comment (idcuadrowolox) [reply maquetado/scss/_footer.scss]: In this case, instead of 60px should be 400px

File: maquetado/scss/_footer.scss [hunk 2]
```diff
@@ -14,9 +14,9 @@
   }
 
   &-logo {
-    height: 60px;
+    height: auto;
```
Comment (henryzarza) [inline maquetado/scss/_footer.scss]: This is the default value, you can delete it

**Review by henryzarza — APPROVED** [review]

**Pull request merged**